src/test/java/uk/gov/digital/ho/pttg/api/IncomeSummaryServiceTest.java

```diff
@@ -118,34 +115,6 @@ public void shouldAllowOptionalToDate() {
         then(mockHmrcClient).should().populateIncomeSummary(eq(SOME_ACCESS_CODE), eq(SOME_INDIVIDUAL), eq(SOME_FROM_DATE), isNull(), any(IncomeSummaryContext.class));
     }
 
-    @Test
-    public void shouldAudit() {
```

Comment:
This behaviour is still present so I think this test should remain as it still "shouldAudit".